Relax version requirements for py-setuptools to enable build with older
versions of it.

--- pyproject.toml.orig	2026-10-29 18:55:55 UTC
+++ pyproject.toml
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools>=77.0.3"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
